src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/maintenance.html.twig

```diff
@@ -0,0 +1,80 @@
+{#**
+ * 2007-2017 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2017 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain "Admin.Shopparameters.Feature" %}
+
+{%
+    set generalForm = form.general
+%}
+
+{% block content %}
+    <div class="container">
+        {{ form_start(form, {'attr' : {'class': 'form'} }) }}
+        <div class="row">
+            {% block maintenance_form_general %}
+            <div class="col">
+                <div class="card">
+                    <h3 class="card-header">
+                        <i class="material-icons">business_center</i> {{ 'General'|trans }}
```

Comment:
Does the trans here means the 'General' string will be placed under the "Admin.Shopparameters.Feature" default domain ? In Crowdin, this string is located in the "Admin/Global" domain.

Reply:
Yes, you got it. To be modified!